### DISCUSSION TITLE
cloud-game in docker

### DISCUSSION THREAD

**Discussion by Imantsg** [giongto35/cloud-game#359] [Q&A] [answered]
I cloned repository using this command

git clone https://github.com/giongto35/cloud-game.git

Then I run cloud-game in docker using this command provided in instrucyions

make dev.run-docker

I see in console log that it detected my roms

cloud-game-local | --------------------------------------------
cloud-game-local | --- The Library of ROMs                  ---
cloud-game-local | --------------------------------------------
cloud-game-local |     Contra (Contra.nes)
cloud-game-local |     Super Mario Bros (Super Mario Bros.nes)
cloud-game-local | --------------------------------------------
cloud-game-local | --- ROMs: 002                   88.525µs ---
cloud-game-local | --------------------------------------------

when I connect to http://localhost:8000/ I see no games in list.
Do I need to configure something more to see game list?

My OS is Ubuntu 21.10

**Comment by sergystepanov** [comment, marked as answer]
No, it should work. I just rechecked and it is working.
Check the logs in the browser and Docker. Try to run the app without any modifications. Use Chrome. Make sure WebRTC is not blocked in the browser or in the system as well as ports defined here https://github.com/giongto35/cloud-game/blob/master/docker-compose.yml. Then try host mode in the docker-compose.

**Comment by Imantsg** [comment]
Yes it looks like it did not work in Firefox.
